src/wallet/rpcwallet.cpp

```diff
@@ -1560,7 +1560,7 @@ UniValue listreceivedbyaddress(const JSONRPCRequest& request)
             "    \"confirmations\" : n,               (numeric) The number of confirmations of the most recent transaction included\n"
             "    \"label\" : \"label\",               (string) The label of the receiving address. The default label is \"\".\n"
             "    \"txids\": [\n"
-            "       n,                                (numeric) The ids of transactions received with the address \n"
+            "       \"txid\",                         (string) The ids of transactions received with the address \n"
```

Comment:
Bad alignment.

Reply:
Remove whitespace at end of line.

Reply:
sorry, I didn't get which part is bad. You mean `,` should be in the next line?

Reply:
```
Result:
[
  {
    "involvesWatchonly" : true,        (bool) Only returned if imported addresses were involved in transaction
    "address" : "receivingaddress",  (string) The receiving address
    "account" : "accountname",       (string) DEPRECATED. Backwards compatible alias for label.
    "amount" : x.xxx,                  (numeric) The total amount in BTC received by the address
    "confirmations" : n,               (numeric) The number of confirmations of the most recent transaction included
    "label" : "label",               (string) The label of the receiving address. The default label is "".
    "txids": [
       "txid",                         (string) The ids of transactions received with the address
       ...
    ]
  }
  ,...
]
```
I mean the alignment of the descriptions but there are others that need to be fixed. Just remove the space. (feel free to fix the alignment of the other lines)